third_party/rapidcheck.BUILD

```diff
@@ -0,0 +1,23 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+cmake(
+    name = "rapidcheck",
```

Comment:
I looked at the build. Doesn't seem  especially complicated. Any reason not to use pure bazel?

I prefer we use bazel except when it would be obnoxious (like check).

Reply:
I thought it will be hard to add it but indeed it was straightforward.